solutions/functions/remove-first-alphabet.js

```diff
@@ -0,0 +1,30 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, 'utf-8');
+let idx = 0;
+data = data.split('\n');
+
+function readLine() {
+  idx++;
+  return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+// Write function with name removeFirstAlphabet which takes a list of string
+// Remove first letter from each string and return the list
+
+function removeFirstAlphabet(list){
+
+    for(let i = 0; i < list.length; i++){
+        list[i] = list[i].substring(1);
```

Comment:
Problem description says `If only one letter is present in the string, delete that element from the list.`

@LVSsandeepVarma Does any of the testcases check for this case? e.g. `abc a xyz`

Reply:
@sailesh-ramayanam  yes we have  a test case which contains one letter